src/main/java/org/folio/service/impl/AllowedServicePointsServiceImpl.java

```diff
@@ -49,7 +50,12 @@ public AllowedServicePointsResponse getAllowedServicePoints(AllowedServicePoints
   }
 
   private AllowedServicePointsResponse getForCreate(AllowedServicePointsRequest request) {
-    String patronGroupId = userService.find(request.getRequesterId()).getPatronGroup();
+    String patronGroupId = request.getPatronGroupId();
+    String requesterId = request.getRequesterId();
+    log.info("getForCreate:: requester patron group and id: {}, {}", patronGroupId, requesterId);
```

Comment:
I think this log message is redundant. We log request params in the controller, and then the whole request in `org.folio.service.impl.AllowedServicePointsServiceImpl#getAllowedServicePoints`

Reply:
removed